### DISCUSSION TITLE
Weird quicklook bug

### DISCUSSION THREAD

**Discussion by setanarut** [sbarex/QLMarkdown#29] [Q&A] [answered]
Does this happen to you too? just me?

https://user-images.githubusercontent.com/36481442/103349247-676e1f80-4aad-11eb-9390-38c0858858c2.mov

try these files; https://github.com/hazarek/markdown-render-test
If I switch to full screen while in light mode, it switches to dark theme. Valid for markdown.css and similar css.

**Comment by sbarex** [comment, marked as answer]
happens to me too
